cMake/FreeCAD_Helpers/SetupCoin3D.cmake

```diff
@@ -35,5 +35,5 @@ macro(SetupCoin3D)
     IF(NOT PIVY_VERSION)
       execute_process (COMMAND ${Python3_EXECUTABLE} -c "import pivy as p; print(p.__version__,end='')" OUTPUT_VARIABLE PIVY_VERSION)
     ENDIF()
-
+    
```

Comment:
Clean up trailing whitespace

Reply:
The sooner we get clang-format in CI the better.